.github/workflows/continuous-integration.yaml

```diff
@@ -40,8 +40,8 @@ jobs:
     name: Unit tests
     runs-on: ubuntu-latest
 
-    # max run time 10 minutes
-    timeout-minutes: 10
+    # max run time 12 minutes
```

Comment:
Leaving in this increase as highest runtime I saw was 10 minutes (it varies, but don't want bogus failure just because CI system may be slow).